chat-services/mattermost-gitlab.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+- hash: f2cf7f90f65214143db61c42fb06d0dda3eebaa6
+  hash_length: 7
+  name: 
```

Comment:
Please add names - for `chat-integrations` repository, names should be in a form of `reponame-subdir` - e.g. `chat-integrations-github`

Reply:
okey